organisations/relationship_transfer.go

```diff
@@ -68,20 +68,31 @@ func getNodeRelationshipNames(cypherRunner neoutils.CypherRunner, uuid string) (
 	return relationshipsFromNodeWithUUID, relationshipsToNodeWithUUID, nil
 }
 
+func constructTransferRelationshipsWithPlatformVersionFromNodeQuery(fromUUID string, toUUID string, predicate string) *neoism.CypherQuery {
+	transferAnnotationsQuery := &neoism.CypherQuery{
+		Statement: fmt.Sprintf(`MATCH (oldNode:Organisation {uuid:{fromUUID}})-[oldRel:%s]->(p)
+					WHERE EXISTS(oldRel.platformVersion)
+					MATCH (newNode:Organisation {uuid:{toUUID}})
+					MERGE (newNode)-[newRel:%s{platformVersion:oldRel.platformVersion}]->(p)
+					on create SET newRel = oldRel
```

Comment:
I guess you could count the relationships(annotations) for each uuid and update the node with more relationships with the canonical uuid and transfer the rest. That won't help obviously if the nodes have both lots of relationships.